cnxarchive/sql/search/title.part.sql

```diff
@@ -10,4 +10,4 @@ SELECT
 FROM
   latest_modules
 WHERE
-  name ~* req(%({0})s::text)
+  StripHTML(name) ~* req(%({0})s::text)
```

Comment:
I was considering something similar, but I honestly feel it'd be too much trouble to add an extra index.